docs/snippets/ov_denormals.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from openvino.runtime import Core
+
+device_name = 'CPU'
+xml_path = 'modelWithDenormals.xml'
+# ! [ov:intel_cpu:denormals_optimization:part0]
+core = Core()
+core.set_property({'CPU_DENORMALS_OPTIMIZATION': 'YES'})
```

Comment:
I suppose we need to implement property in the same way https://github.com/openvinotoolkit/openvino/pull/11650/files#diff-73e2aadc8029dd3dcdc2a8793dbe1d984674020a5958a8afad665661163bbfe8R31, how do you think?

Reply:
BTW, I see it's implemented. So, why don't we use `denormals_optimization`?

Reply:
done, thanks!